tools/test/examples/update.py

```diff
@@ -306,20 +383,31 @@ def main(arguments):
     parser.add_argument('tag', help="mbed-os tag to which all examples will be updated")
     parser.add_argument('-c', '--config_file', help="Path to the configuration file (default is 'examples.json')", default='examples.json')
     parser.add_argument('-T', '--github_token', help="GitHub token for secure access")
-    parser.add_argument('-U', '--github_user', help="GitHub user for forked repos")
+    parser.add_argument('-l', '--log-level', 
+                        help="Level for providing logging output", 
+                        default='INFO')
+    
+    exclusive = parser.add_mutually_exclusive_group(required=True)
+    exclusive.add_argument('-U', '--github_user', help="GitHub user for forked repos, mutually exclusive to branch option")
+    exclusive.add_argument('-b', '--branch', help="Branch to be updated, mutually exclusive to user option")
     
     args = parser.parse_args(arguments)
 
-    cfg = os.path.join(os.path.dirname(__file__), args.config_file)
-    
+    default = getattr(logging, 'INFO')
+    level = getattr(logging, args.log_level.upper(), default)
+        
+    # Set logging level
+    logging.basicConfig(level=level)
+
+    update_log = logging.getLogger("Update")
```

Comment:
Should this be a global? have you run the script to verify that you get the correct logger when running the other functions in this file?